### DISCUSSION TITLE
Revamp the hijack objective to be more like how it currently is in /tg/.

### DISCUSSION THREAD

**Discussion by PetMudstone** [unitystation/unitystation#7168] [Ideas]
Right now, the hijack objective succeeds when traitors escape alone on the escape shuttle with no one else alive on said shuttle.

The only solution to the objective is just to murder everyone else on the shuttle. This can prove to be quite difficult for traitors when they have to contend with ten or so people on the shuttle, several of whom are likely armed and armored and even fewer possibly being other armed traitors. Additionally it can be quite difficult to make sure everyone is actually dead on the shuttle. Survivors who are hiding on the shuttle or are unconscious could cause a traitor to fail if they're not thorough enough.

/tg/ gets around these issues by making it so that the hijack objective is complete when a traitor hacks the shuttle console. Traitors hack the shuttle console by alt-clicking it 5 times. Each time the shuttle console is alt clicked there is a progress bar that has to fill up before you can alt-click the console again. Additionally each successful alt-click extends the shuttle timer to give the traitor additional time to hack. Upon completing 5 alt-clicks the hack is complete and the shuttle timer is reduced to 10 seconds. If the hack is complete the objective will succeed regardless of any surviving crew on the shuttle or of if the traitor with the objective is still alive.

The current way /tg/ does hijacking not only makes it so that traitors don't have to worry about killing absolutely everyone on the shuttle, it also gives them more options for completing their objectives. Traitors can put everyone to sleep with gas or barricade themselves in the cockpit in order to hack in peace. Or they can just try to kill everyone again as per usual.

**Comment by wraith-54321** [comment]
also note, the hyjack can be finished after launch
